chef/cookbooks/crowbar-pacemaker/recipes/drbd.rb

```diff
@@ -26,7 +26,10 @@
 
 if claimed_disks.empty? and not unclaimed_disks.empty?
   unclaimed_disks.each do |disk|
-    if disk.claim(claim_string)
+    %x{lsblk #{disk.name} --noheadings --output MOUNTPOINT | grep -q -v ^$}
```

Comment:
That may be a good idea. However, it would conflict with https://github.com/crowbar/barclamp-provisioner/blob/master/chef/cookbooks/provisioner/recipes/bootdisk.rb#L5 which also looks for unclaimed disks that may be already mounted.